angular.json

```diff
@@ -3327,6 +3327,72 @@
           }
         }
       }
+    },
+    "elements": {
+      "projectType": "library",
+      "root": "libs/fluid-elements",
+      "sourceRoot": "libs/fluid-elements",
+      "architect": {
+        "build": {
+          "builder": "@dynatrace/barista-builders:elements",
+          "options": {
+            "releasePackageJson": "libs/fluid-elements/package.json",
+            "outputPath": "dist/libs/fluid-elements",
+            "buildTag": "scope:elements",
+            "assets": [
+              {
+                "glob": "**/*",
+                "input": "libs/fluid-elements/assets",
+                "output": "dist/libs/elements/assets"
+              }
+            ]
+          },
+          "configurations": {
+            "production": {}
```

Comment:
please add configuration that you run on production the target production inside your schedule target